h2o-genmodel/src/main/java/hex/genmodel/utils/ArrayUtils.java

```diff
@@ -0,0 +1,81 @@
+package hex.genmodel.utils;
```

Comment:
ArrayUtils is a very high-traffic class, and moving it to a new place will almost surely break someone's code. Of course we could also do the inheritance trick that you suggested previously, but we'd better have a discussion about that first (and in general a discussion about the structure of the genmodel package)...